packages/prisma-types/src/zod-schemas.ts

```diff
@@ -33,7 +33,11 @@ export const channelBitfieldFlags = [
 
 export const zChannelBitfieldFlags = toZObject(...channelBitfieldFlags);
 
-export const zChannel = z.object({
+type ChannelZodFormat = {
+  [K in keyof Channel]: z.ZodTypeAny;
+};
+
+export const zChannelPrisma = z.object({
```

Comment:
Look into using https://github.com/jbranchaud/til/blob/master/zod/incorporate-existing-type-into-zod-schema.md for this